mindsdb/integrations/handlers/rockset_integration/rockset_handler.py

```diff
@@ -0,0 +1,7 @@
+from ..mysql_handler import Handler as MySQLHandler, connection_args, connection_args_example
+
+class RocksetIntegration(MySQLHandler):
+    """
+    This handler handles connection and execution of the Rockset integration
+    """
+    name = 'rockset'
```

Comment:
We need the `__init__` function here. Please check https://github.com/mindsdb/mindsdb/blob/b830736ae6e822095992e8f924efb0b1b47a4ebe/mindsdb/integrations/handlers/mariadb_handler/mariadb_handler.py#L10

Reply:
OK, I put it back